Packages.props

```diff
@@ -17,7 +17,7 @@
 		<PackageReference Update="NUnit3TestAdapter" Version="[3.17.0]" />
 		<PackageReference Update="SimpleLogInterface" Version="[3.0.1]" />
 		<PackageReference Update="System.Collections.Immutable" Version="[5.0.0]" />
-		<PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="[6.10.0]" />
+		<PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="[6.10.1]" />
```

Comment:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/compare/6.10.0...6.10.1